psi4/src/export_fock.cc

```diff
@@ -94,7 +94,9 @@ void export_fock(py::module &m) {
         .def("K", &JK::K, py::return_value_policy::reference_internal)
         .def("wK", &JK::wK, py::return_value_policy::reference_internal)
         .def("D", &JK::D, py::return_value_policy::reference_internal)
-        .def("print_header", &JK::print_header, "docstring");
+        .def("print_header", &JK::print_header, "docstring")
+        .def("do_incfock_iter", &JK::do_incfock_iter, "Was the last Fock build incremental?")
```

Comment:
I think the do_incfock_iter function could be renamed for clarity. Something like last_iter_incfock or incfock_last_iter or something along those lines.